modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/higher_level_reviews_controller.rb

```diff
@@ -102,7 +102,7 @@ def new_higher_level_review
       auth_headers: request_headers,
       form_data: @json_body,
       source: request_headers['X-Consumer-Username'],
-      api_version: 'V2'
+      api_version: 'v2'
```

Comment:
This might become an issue since we'll have existing records with `V2` (upcase) but new records with `v2` (downcase) - things like the [HLR version scopes](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/app/models/appeals_api/higher_level_review.rb#L22-L28) would only return a subset of the records since it's case-sensitive.

I think the easiest thing to do would be to default to upcase instead of downcase to avoid this scoping issue.